### ISSUE TITLE
[$20][MSFT-207] [Member] -  The Text is not readable in Footer Menu after getting highlighted.

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2870]
**Steps to Reproduce**
1. Go to https://accounts.topcoder.com/member
2. Go to the Footer. 
3. Highlighted the Menu listed. 
4. Observe the content. 

**Expected Result**
The Text under footer Menu should be readable and clear even after highlighting.

**Actual Result**
The text appears to be unreadable after getting highlighted, as the highlighted color theme appears to be darker than that of the text color.

**Device**: Desktop/Labtop
**Operating System**: macOS
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: AAA
**WCAG Success Criterion**: 1.4.8 Visual Presentation
**Screen Reader Used**: 
**Accessibility Audit Tool used**: Other

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=63cedfe6%2D52a5%2D4f33%2D9399%2D00e26b7dd4a2&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2FCreeya%2FBug%20%2D%20Footer%20text

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096249 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096249 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096249 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by asadath1395** [comment]
@veshu @mishacucicea This is part of home page and out of scope. Please confirm

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096249 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by mishacucicea** [comment]
@asadath1395 No, it's still about the login, I corrected the link, it had `/members` instead of `/member`.

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096249 has been updated - it has been assigned to Timor.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by romitgit** [comment]
@mishacucicea Which repo is this? Where is the code for this one?

**Comment by veshu** [comment]
@romitgit This is related to home page, removing from scope for now.

cc @mishacucicea please correct if I get wrong.

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096782 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by crazyk07** [comment]
This ticket was not processed for payment. If you would like to process it for payment, please reopen it, add the ```tcx_FixAccepted``` label, and then close it again<br/><br/>```This is an automated message for crazyk via Topcoder X```
